code/game/objects/items/stacks/medical.dm

```diff
@@ -127,6 +127,23 @@
 	self_delay = 20
 	max_amount = 12
 
+/obj/item/stack/medical/gauze/wirecutter_act(mob/living/user, obj/item/I)
+	if(get_amount() < 2)
+		to_chat(user, "<span class='warning'>You need at least two gauze to do this!</span>")
+		return
+
+	else
+		var/obj/item/stack/sheet/cloth/new_item = new(usr.loc)
+		user.visible_message("[user.name] cut [src] into pieces of cloth with [I].", \
+					 "<span class='notice'>You cut [src] into pieces of cloth with [I].</span>", \
+					 "<span class='italics'>You hear cutting.</span>")
+		var/obj/item/stack/medical/gauze/R = src
+		src = null
+		var/replace = (user.get_inactive_held_item()==R)
+		R.use(2)
+		if (!R && replace)

```

Comment:
Need to check `QDELETED(R)`, `R` won't necessarily go null just from being qdel'd.